ynr/apps/elections/templates/elections/ballot_view.html

```diff
@@ -91,15 +94,23 @@ <h1>Candidates for {{ ballot.post.label }}</h1>
               </a>
             </td>
             <td>{{ candidate.party.name }}</td>
+             
+            {% for membership in candidate.person.memberships.all %}
```

Comment:
In this context, `candidate` is the membership object, so you can use it to get the `previous_party_affiliations` directly:
```
for party in membership.previous_party_affiliations.all
....
```

Reply:
As we dont want to go through every candidacy for a person and get the previous party affiliations, we only display the ones that are related to this candidacy